bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_NLO_fxfx/Wgamma_Monophoton_Ptg130_NLO_run_card.dat

```diff
@@ -0,0 +1,152 @@
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1 =  run_tag  ! name of the run
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+        100 = nevents
```

Comment:
Better to have few K events for pilot run (3000 will be fine)

Reply:
Changed